test-file-trees/bad-structure/cli/invalid-data-accepts/invalid-data-accepts.spec.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	description: 'Order a pizza, to-go',
+	data: {
+		description: 'What type of pizza to order',
+		type: 'fake',
```

Comment:
Why is this set to `fake`? The directory name you chose implies that the problem is an invalid `accepts` value, but here you're testing an invalid `type` value...

Reply:
Yeah, I was seeing if I could combine multiple test cases using the same file, but it's fine.  Single test is good.